docs/src/man/reshaping_new.md

```diff
@@ -0,0 +1,337 @@
+   3 │ male    Age       49
+   4 │ male    Age       45
+   5 │ male    Age       53
+   6 │ male    Age       35
+   7 │ male    Age       53
+   8 │ male    Age       35
+  ⋮  │   ⋮        ⋮        ⋮
+  24 │ male    Housing   free
+  25 │ male    Housing   free
+  26 │ male    Housing   free
+  27 │ male    Housing   own
+  28 │ male    Housing   rent
+  29 │ male    Housing   own
+  30 │ male    Housing   own
+                15 rows omitted
+```
+
+Below we have omitted second argument in `stack`, so all other columns are assumed to be the
+[id_variables](https://github.com/JuliaData/DataFrames.jl/blob/f690aa49e958f51e0c3c579b6def1f11be214d98/src/abstractdataframe/reshape.jl#:~:text=id_vars%60%20%3A%20the%20identifier,are%20not%20%60measure_vars%60):
+
+```jldoctest dataframe
+julia> stack(german, Not([:Sex, :id]))
+40×4 DataFrame
+ Row │ id     Sex     variable         value
+     │ Int64  String  String           Any
+─────┼────────────────────────────────────────────
+   1 │     0  male    Age              67
+   2 │     1  female  Age              22
+   3 │     2  male    Age              49
+   4 │     3  male    Age              45
+   5 │     4  male    Age              53
+   6 │     5  male    Age              35
+   7 │     6  male    Age              53
+   8 │     7  male    Age              35
+  ⋮  │   ⋮      ⋮            ⋮             ⋮
+  34 │     3  male    Saving accounts  little
+  35 │     4  male    Saving accounts  little
+  36 │     5  male    Saving accounts  missing
+  37 │     6  male    Saving accounts  quite rich
+  38 │     7  male    Saving accounts  little
+  39 │     8  male    Saving accounts  rich
+  40 │     9  male    Saving accounts  little
+                                   25 rows omitted
+```
+
+We can rename columns:
+
+```jldoctest dataframe
+julia> stack(german, Not([:Sex, :id]), variable_name=:x, value_name=:y)
+40×4 DataFrame
+ Row │ id     Sex     x                y
+     │ Int64  String  String           Any
+─────┼────────────────────────────────────────────
+   1 │     0  male    Age              67
+   2 │     1  female  Age              22
+   3 │     2  male    Age              49
+   4 │     3  male    Age              45
+   5 │     4  male    Age              53
+   6 │     5  male    Age              35
+   7 │     6  male    Age              53
+   8 │     7  male    Age              35
+  ⋮  │   ⋮      ⋮            ⋮             ⋮
+  34 │     3  male    Saving accounts  little
+  35 │     4  male    Saving accounts  little
+  36 │     5  male    Saving accounts  missing
+  37 │     6  male    Saving accounts  quite rich
+  38 │     7  male    Saving accounts  little
+  39 │     8  male    Saving accounts  rich
+  40 │     9  male    Saving accounts  little
+                                   25 rows omitted
+```                                           
+
+## The basics of `unstack`
+
+*Unstack* converts the data frame `german` from it long to wide format. Row and column 
```

Comment:
you do not unstack `german` below. Also the order of column keys is not guaranteed to be in order of appearance in general